src/modules/auto-google-map.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Auto Google Map module
+ * @author Deux Huit Huit
+ * @requires com/google-map.js
+ */
+(function ($, undefined) {
+
+	'use strict';
+
+	var scope = $('.page');
+
+	var sels = {
+		ctn: '.js-auto-google-map-ctn',
+		map: '.js-auto-google-map',
+		marker: '.js-auto-google-map-marker',
+		styles: '.js-auto-google-map-style'
+	};
+
+	var ready = false;
+
+	var DEFAULT_MARKER_WIDTH = 30;
+	var DEFAULT_MARKER_HEIGHT = 30;
+	var DEFAULT_ZOOM = 10;
+
+	var createMarker = function (element, mapComponent) {
+		var lat = element.attr('data-lat');
+		var lng = element.attr('data-lng');
+		var iconSrc = element.attr('data-icon');
+		var iconWidth = parseInt(element.attr('data-icon-width'), 10) || DEFAULT_MARKER_WIDTH;
+		var iconHeight = parseInt(element.attr('data-icon-height'), 10) || DEFAULT_MARKER_HEIGHT;
+
+		if (!!App.device.internetexplorer) {
+			iconSrc = null;
```

Comment:
why ?

Reply:
I'm not sure, but I think it has to do with svg images as pins.

Reply:
Ping @nitriques

Reply:
Well @fhamon  @PascalPiche we need to be sure.... no ?